app/src/main/java/com/example/beerapp/BeerListRepository.java

```diff
@@ -1,8 +1,6 @@
 package com.example.beerapp;
 
-import retrofit2.Retrofit;
-
 public interface BeerListRepository {
 
-    Retrofit getNetworkData();
+    void getNetworkData();
```

Comment:
the better name is `loadBeers()` or `getBeers()`. Its only interface not an implementation, so you shouldn`t use words such as network, remote, local